web/src/components/SocialMediaTooltip.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { ReactElement, ReactNode } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled, { css } from 'styled-components'
+
+import { getSlugFromPath, POIS_ROUTE } from 'api-client'
+import { UiDirectionType } from 'translations'
+
+import { CloseIcon, FacebookIcon, MailSocialIcon, WhatsappIcon } from '../assets'
+import dimensions from '../constants/dimensions'
+import useWindowDimensions from '../hooks/useWindowDimensions'
+import { RouteType } from '../routes'
+import Tooltip from './Tooltip'
+
+type SocialMediaTooltipProps = {
+  active: boolean
+  children: ReactNode
+  shareLink: string
+  title: string
+  onClose: () => void
+  flow: 'top' | 'right'
+  direction: UiDirectionType
+  route: RouteType
+}
+
+const SHARE_BUTTON_WIDTH = 50
+const TooltipContainer = styled.div<{
+  flow: 'top' | 'right'
```

Comment:
I think we should use `horizontal` and `vertical` as `right` is left on RTL layouts.